spec/acceptance/xtypes/jenkins_credentials_spec.rb

```diff
@@ -52,6 +52,7 @@
           # Run it twice and test for idempotency
           apply(pp, :catch_failures => true)
           apply(pp, :catch_failures => true)
+          #apply(pp, :catch_failures => true)
```

Comment:
It's actually just to test that master is working. In fact, if you check my first commit, I initially tried just to comment the second apply as that was the error I got within PR #583 (failed on line 54 in the beaker test). And now, the test even don't start